docs/gcp-config.md

```diff
@@ -34,6 +34,8 @@ To integrate Heptio Ark with GCP, create an Ark-specific [Service Account][15]:
         --display-name "Heptio Ark service account"
     ```
 
+> If you'll be using Ark to backup multiple clusters with multiple GCS buckets, it may be desirable to create a unique username per cluster rather than the default `heptio-ark`.
+
     Then list all accounts and find the `heptio-ark` account you just created:
```

Comment:
The formatting starting here is messed up (it's not immediately apparent to me why) -- see https://github.com/metadave/ark/blob/c2d65ae0bd35154f5f7bb6daf493824393917e97/docs/gcp-config.md#create-service-account -- "Then list ... its `email` value" is all within a code block.

Reply:
I needed to tab the `>` over.